tests/test_markup_utils.py

```diff
@@ -0,0 +1,171 @@
+# (c) 2012-2019, Ansible by Red Hat
```

Comment:
Please don't mix styles. You're using pytest and xUnit styles together in one test file. Please stick to one at least in file scope.